README.rst

```diff
@@ -133,6 +138,61 @@ Or in Python::
 
     cs.listVirtualMachines(fetch_list=True)
 
+Async client
+------------
+
+``cs`` provides the ``AIOCloudStack`` class for async/await calls in Python
+3.5+.
+
+.. code-block:: python
+
+    from cs import AIOCloudStack, read_config
+
+    cs = AIOCloudStack(**read_config())
+    vms = await cs.listVirtualMachines()
+
+By default, this client polls CloudStack's async jobs to return actual results
+for commands that result in an async job being created. You can customize this
+behavior with ``job_timeout`` (default: None -- wait indefinitely) and
+``poll_interval`` (default: 2s).
+
+.. code-block:: python
+
+    cs = AIOCloudStack(**read_config(), job_timeout=300, poll_interval=5)
+
+Async deployment of multiple vms
+________________________________
+
+.. code-block:: python
+
+    import asyncio
+    from cs import AIOCloudStack, read_config
+    acs = AIOCloudStack(**read_config('preprod:testing'))
```

Comment:
`**read_config()`

Not sure it's worth calling the async client differently (first async example uses `cs`)